tests/contracts/hello/src/lib.rs

```diff
@@ -0,0 +1,191 @@
+#![feature(wasm_abi)]
```

Comment:
Since this code currently also serves as an example contract (as there's no other documentation) i think it might make sense adding unit tests for it, as an example on how to test the contract with unit tests. 

Not immediately clear to me, how hard it would be to have a simple test framework for testing contract calls locally, might make more sense deferring this to a followup work.

Reply:
Added a simple way to test smart contracts in unit tests, please take a look.